composer.json

```diff
@@ -94,6 +94,7 @@
             "@composer normalize --working-dir=dev-tools --dry-run ../composer.json",
             "@composer normalize --working-dir=dev-tools --dry-run composer.json"
         ],
+        "php-compatibility": "@php dev-tools/vendor/bin/phpcs -p --standard=dev-tools/php-compatibility/phpcs-php-compatibility.xml",
```

Comment:
how does it work? shall we run it on each supported PHP version?

Reply:
No, there is [config](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/7844/files#diff-835bd72f25290942738449f23f502f9709f9866cc1a7bf3232b687d26fb7efdeR22) which defines range of PHP versions project must be compatible with. The sniffs just check the code and verify if there's not anything which won't work on any of supported PHP versions.